ext/opcache/config.w32

```diff
@@ -23,7 +23,9 @@ if (PHP_OPCACHE != "no") {
 	ADD_EXTENSION_DEP('opcache', 'pcre');
 
 	if (PHP_OPCACHE_JIT == "yes") {
-		if (CHECK_HEADER_ADD_INCLUDE("ir/ir.h", "CFLAGS_OPCACHE", PHP_OPCACHE + ";ext\\opcache\\jit")) {
+		if (TARGET_ARCH == 'arm64') {
+			WARNING("JIT not enabled; not yet supported for ARM64");
```

Comment:
Based on https://github.com/dstogov/ir/tree/master/dynasm I was under the impression ARM is supported, so it isn't or only ARM on Windows?

Reply:
Supported is only aarch64 on POSIX systems. See https://github.com/php/php-src/issues/16839#issuecomment-2481472101 f.

Reply:
Perhaps it might be wise to specify in the warning message that JIT is not support on ARM64 on Windows (indicating that it is supported on POSIX systems)

Reply:
This file is part of the Windows build chain; it is not used for building on POSIX systems.

Reply:
Well yeah, but the warning is meant for humans

Reply:
````
$ configure --with-toolset=gcc
PHP Version: 8.5.0-dev

Saving configure options to config.nice.bat
ERROR: Unsupported toolset 'gcc'
````

Reply:
My point was that the warning could be misinterpreted by windows-only developers who don't know much about JIT, leding them to think that JIT is not supported at all on arm64, when they could be instead directed to use WSL, where JIT is available, but ah well :)